pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -190,6 +191,9 @@ class ModelCheckpoint(Callback):
 
         *monitor, mode, every_n_train_steps, every_n_epochs, train_time_interval, save_on_train_epoch_end*
 
+    .. note:: If the checkpoint's ``dirpath`` changed from what it was before while resuming the training,
+        only ``last_model_path`` and ``best_model_path`` will be tracked and a warning will be issued.
```

Comment:
can we add it to the existing Note section above?

Reply:
And I fail to see why "tracking" the remaining attributes would suddenly be prevented. 
It's not loading them, that's clear. But we can still encounter a new best model and start with the new top-k if we want.

Reply:
Is it because we don't load best_model_score?

Reply:
> can we add it to the existing Note section above?

Done! :)

> And I fail to see why "tracking" the remaining attributes would suddenly be prevented.

Sorry, `tracking` was the wrong term to use. In the follow-up PR, to fix the bug you also mentioned, we'll take a look at what should be reloaded, and what not.